### PR TITLE
geth: add new update transitions helper utility

### PR DESCRIPTION
**Pull request by rodion-lim-partior**
[ADDED] Create new command to update transitions, e.g. block period seconds in future transition block

### CODE REVIEW

**Review by frank-lim-partior — DISMISSED** [review]
Code was lifted from geth init.

Integration testing including helm changes that has call to this API.
- all nodes transited correctly at the target block without issues.

File: cmd/geth/chaincmd.go [hunk 1]
```diff
@@ -206,6 +221,64 @@ func getIsQuorum(file io.Reader) bool {
 	return altGenesis.Config.IsQuorum == nil || *altGenesis.Config.IsQuorum
 }
 
+// updateTransitions will update genesis block with the new transitions data
+func updateTransitions(ctx *cli.Context) error {
+	// Open and initialise both full and light databases
```
Comment (frank-lim-partior) [inline cmd/geth/chaincmd.go]: recommend moving this database opening to right before the forloop for updating the DBs

File: cmd/geth/chaincmd.go [hunk 2]
```diff
@@ -206,6 +221,64 @@ func getIsQuorum(file io.Reader) bool {
...
+		// Check that new transitions have changed before updating them
+		if !reflect.DeepEqual(storedcfg.Transitions, genesis.Config.Transitions) {
+			log.Info("Change found in transitions, proceeding to update chain config")
+			storedcfg.Transitions = genesis.Config.Transitions
+			rawdb.WriteChainConfig(chaindb, stored, storedcfg)
+		} else {
+			log.Info("No change in transitions, no update required to chain config")
+		}
+	}
+	return nil
```
Comment (frank-lim-partior) [inline cmd/geth/chaincmd.go:278]: Perhaps we should return 'true' if a change was made, and false if nothing. If this goes back to the api caller.

File: cmd/geth/chaincmd.go (hunk 1 above)
Comment (rodion-lim-partior) [reply cmd/geth/chaincmd.go]: amended

**Review by frank-lim-partior — APPROVED** [review]

File: cmd/geth/chaincmd.go (hunk 2 above)
Comment (frank-lim-partior) [reply cmd/geth/chaincmd.go:278]: not modifying as this is a nice-to-have that might disrupt geth api wrapping - to investigate separately.

Comment (rodion-lim-partior) [reply cmd/geth/chaincmd.go:278]: rest of the chainCmd and accountCmd only returns an error, suggest we keep it consistent for now. Making a change to the interface requires an entire round of regression testing, will revisit this in future

**Review by john-sobrepena-partior — APPROVED** [review]

**Pull request merged**